src/ol/tile.js

```diff
@@ -4,6 +4,7 @@ goog.require('ol');
 goog.require('ol.events.EventTarget');
 goog.require('ol.events.EventType');
 
+var lastSequenceNumber = 0;
```

Comment:
Yes, that's fine. We're doing it [elsewhere](https://github.com/openlayers/ol3/blob/v3.18.1/src/ol/object.js#L105-L109) already, and adding that comment to `ol.getUid()` is overdue.